Marlin/src/lcd/language/language_vn.h

```diff
@@ -0,0 +1,1506 @@
+#endif
+#ifndef MSG_DELTA_CALIBRATE_CENTER
+// Calibrate Center
+  #define MSG_DELTA_CALIBRATE_CENTER          _UxGT("Chỉnh Z Center")
+#endif
+#ifndef MSG_DELTA_SETTINGS
+// Delta Settings
+  #define MSG_DELTA_SETTINGS                  _UxGT("Cài Đặt Delta")
+#endif
+#ifndef MSG_DELTA_AUTO_CALIBRATE
+// Auto Calibration
+  #define MSG_DELTA_AUTO_CALIBRATE            _UxGT("Cân Chỉnh Tự Động")
+#endif
+#ifndef MSG_DELTA_HEIGHT_CALIBRATE
+// Set Delta Height
+  #define MSG_DELTA_HEIGHT_CALIBRATE          _UxGT("Đặt Chiều Cao Delta")
+#endif
+#ifndef MSG_DELTA_Z_OFFSET_CALIBRATE
+// Probe Z-offset
+  #define MSG_DELTA_Z_OFFSET_CALIBRATE        _UxGT("Đầu Dò Z-Bù Đắp")
+#endif
+#ifndef MSG_DELTA_DIAG_ROD
+// Diag Rod
+  #define MSG_DELTA_DIAG_ROD                  _UxGT("Gậy Chéo")
+#endif
+#ifndef MSG_DELTA_HEIGHT
+// Height
+  #define MSG_DELTA_HEIGHT                    _UxGT("Chiều Cao")
+#endif
+#ifndef MSG_DELTA_RADIUS
+// Radius
+  #define MSG_DELTA_RADIUS                    _UxGT("Bán Kính")
+#endif
+#ifndef MSG_INFO_MENU
+  #define MSG_INFO_MENU                       _UxGT("Về Máy In")
+#endif
+#ifndef MSG_INFO_PRINTER_MENU
+// Printer Info
+  #define MSG_INFO_PRINTER_MENU               _UxGT("Thông Tin Máy In")
+#endif
+#ifndef MSG_3POINT_LEVELING
+// 3-Point Leveling
+  #define MSG_3POINT_LEVELING                 _UxGT("San lấp 3-Điểm")
+#endif
+#ifndef MSG_LINEAR_LEVELING
+// Linear Leveling
+  #define MSG_LINEAR_LEVELING                 _UxGT("San Lấp Tuyến Tính")
+#endif
+#ifndef MSG_BILINEAR_LEVELING
+// Bilinear Leveling
+  #define MSG_BILINEAR_LEVELING               _UxGT("San Lấp Song Tuyến")
+#endif
+#ifndef MSG_UBL_LEVELING
+// Unified Bed Leveling
+  #define MSG_UBL_LEVELING                    _UxGT("San Lấp Bàn Thống Nhất")
+#endif
+#ifndef MSG_MESH_LEVELING
+// Mesh Leveling
+  #define MSG_MESH_LEVELING                   _UxGT("Lưới San Lấp")
+#endif
+#ifndef MSG_INFO_STATS_MENU
+// Printer Stats
+  #define MSG_INFO_STATS_MENU                 _UxGT("Thống Kê Máy In")
+#endif
+#ifndef MSG_INFO_BOARD_MENU
+// Board Info
+  #define MSG_INFO_BOARD_MENU                 _UxGT("Thông Tin Bo Mạch")
+#endif
+#ifndef MSG_INFO_THERMISTOR_MENU
+// Thermistors
+  #define MSG_INFO_THERMISTOR_MENU            _UxGT("Điện Trở Nhiệt")
+#endif
+#ifndef MSG_INFO_EXTRUDERS
+// Extruders
+  #define MSG_INFO_EXTRUDERS                  _UxGT("Máy đùn")
+#endif
+#ifndef MSG_INFO_BAUDRATE
+// Baud
+  #define MSG_INFO_BAUDRATE                   _UxGT("Baud")
+#endif
+#ifndef MSG_INFO_PROTOCOL
+// Protocol
+  #define MSG_INFO_PROTOCOL                   _UxGT("Giao Thức")
+#endif
+#ifndef MSG_CASE_LIGHT
+// Case light
+  #define MSG_CASE_LIGHT                      _UxGT("Đèn Khuông")
+#endif
+#ifndef MSG_CASE_LIGHT_BRIGHTNESS
+// Light Brightness
+  #define MSG_CASE_LIGHT_BRIGHTNESS           _UxGT("Độ Sáng")
+#endif
+#if LCD_WIDTH >= 20
+  #ifndef MSG_INFO_PRINT_COUNT
+// Print Count
+    #define MSG_INFO_PRINT_COUNT              _UxGT("Số In")
```

Comment:
Print count can be translate to:"Số lần đã in"

Reply:
I think 'Số In' is fine here.